src/components/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -243,15 +247,83 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
     );
   };
 
+  const ButtonFocusUnderline = styled(Svg)<{ $color: OakColorName }>`
+    color: ${(props) => props.$color};
+  `;
+
+  /**
```

Comment:
Think you can delete this comment with changes below

Reply:
Again bad habit 😅